### DISCUSSION TITLE
Create Server on python

### DISCUSSION THREAD

**Discussion by leandrogmuller** [SAP/PyRFC#257] [Q&A] [answered]
Hello.
I need to create a rfc Server that SAP abap stack Will call my Server.
Does it possible?

**Comment by bsrdjan** [comment, marked as answer]
Hi @leandrogmuller, the PyRFC documentation has few pending updates and server functionality documentation is still missing fully supported. Please refer to these docs/examples:

- Overview how the server works
https://github.com/SAP/node-rfc/blob/main/doc/usage.md#server

- Examples of PyRFC server called, to be called from ABAP system
https://github.com/SAP/PyRFC/blob/main/examples/serverStfcConnection.py

Hope this helps until PyRFC docs updated

**Reply by ruediman** [reply]
@leandrogmuller: does the documentation help or is it the server, you're struggling with?

**Reply by leandrogmuller** [reply]
> @leandrogmuller: does the documentation help or is it the server, you're struggling with?

Hi @ruediman , I still haven't time to read documentation. I will read after to finish my project Stack Python Platform because I need to implement RFC server to receive conection from SAP Stack ABAP.
@bsrdjan thanks your answer, I will check.
